docs/views/view_template.rst

```diff
@@ -1,26 +1,50 @@
-.. reference for this view-component
-.. you can refer to this component using :ref:`<component_name>-label`
+.. Refer to a component using :ref:`<component_name>-label`
 
-.. _<view_name>-label:
+.. _view_component_template-label:
```

Comment:
This should actually be copied to tool_interface.rst file, above the `include` command. You can provide example here and add instructions for a new user (i.e. cut those lines and copy them to tool_interface.rst, then replace <view component-name> by the name of your choice